hushline/static/vendor/tesseract.min.js

```diff
@@ -0,0 +1,3 @@
+/*! For license information please see tesseract.min.js.LICENSE.txt */
```

Comment:
According to the docs for tesseract we may want to load it via CDN (and not include it in the vendor files here) because this script further loads CDN hosted resources, https://github.com/naptha/tesseract.js/blob/master/docs/local-installation.md.

Reply:
done